src/components/reusables/Input.tsx

```diff
@@ -25,6 +25,7 @@ export default class Input extends React.Component<InputProps, {}> {
           id={this.props.name}
           defaultValue={this.props.value}
           readOnly={false}
+          defaultChecked={this.props.type === "checkbox" && this.props.value}
```

Comment:
This is fine but it makes more sense to have a `checked` prop rather than use the `value` prop for a checkbox.